datafog/processing/text_processing/spacy_pii_annotator.py

```diff
@@ -12,21 +31,18 @@ class SpacyPIIAnnotator(BaseModel):
 
     @classmethod
     def create(cls) -> "SpacyPIIAnnotator":
-        try:
-            # Try loading as a spaCy model first
-            import spacy
+        import spacy
 
-            nlp = spacy.load("en_spacy_pii_fast")
+        try:
+            nlp = spacy.load("en_core_web_lg")
         except OSError:
-            # If that fails, try importing as a module
-            try:
-                import en_spacy_pii_fast
-
-                nlp = en_spacy_pii_fast.load()
-            except ImportError:
-                raise ImportError(
-                    "Failed to load en_spacy_pii_fast. Make sure it's installed correctly."
-                )
+            import subprocess
+
+            subprocess.run(
+                ["python", "-m", "spacy", "download", "en_core_web_lg"], check=True
```

Comment:
you should figure out if python or python3 needs to be called here. I know this won't work on my system.

Reply:
Might be able to find the binary running this file and using that.

Reply:
the usage I've seen referenced and also used online has been python, not python3.   is it just a matter of adding handling here for python vs python3?

I ended up changing this command to download the binary directly, not sure if it directly addresses the first concern though..

Reply:
This calls out to the OS. If python doesn't exist this will fail.
I'd suggest this:
import sys

interpreter_location = sys.executable

This statement is helpful:
On all platforms, passing [sys.executable](https://docs.python.org/3/library/sys.html#sys.executable) is the recommended way to launch the current Python interpreter again, and use the -m command-line format to launch an installed module.
https://docs.python.org/3/library/subprocess.html

Reply:
thanks - just implemented that in 924b47b87c50f02104458f7e61f7e41d2e7c5f26
and all checks have passed!